enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -260,18 +271,15 @@ func (r *runnerService) createAction(ctx context.Context, req *rnpb.RunRequest,
 	if err != nil {
 		return nil, status.WrapError(err, "upload command")
 	}
+	// Prefer to set the timeout in the ci_runner, which gracefully shuts down
+	// the runner. Add some buffer before the action is forcefully terminated by
+	// the executor.
+	actionTimeout := *timeout + timeoutGracePeriod
```

Comment:
(if accepting the above suggestion)

```suggestion
	actionTimeout := timeout + timeoutGracePeriod
```